Dockerfile

```diff
@@ -1,8 +1,10 @@
 FROM node:16-alpine
 
+RUN apk add --no-cache git
+
 WORKDIR /app
-ADD . /app
+COPY . /app/
 
-RUN apk add --no-cache git
-RUN yarn install
-ENTRYPOINT yarn start
+RUN yarn install --prod --frozen-lockfile \
```

Comment:
Removing git at this stage won't make the overall image any smaller I believe, since it's a new layer and the layer including git has already been made.

Options to not have git in the resulting image include putting the apk add on the same line as the yarn stuff, or making a new FROM afterwards and copying the generatd build output into it.

Reply:
Fair point.